README.md

```diff
@@ -499,6 +500,9 @@ SAAS
   * IAAS
   * PAAS
   * SAAS</summary><br><b>
+  * IAAS - Users have control over complete Operating System and need not to worry about the physical resources, which is managed by Cloud Service Provider.
+  * PAAS - CLoud Service Provider takes care of Operating System, Middlewares and users only need to focus on our Data and Application.
```

Comment:
s/CLoud/Cloud/
s/focus on our Data and Application/focus on their application and its data/